### PR TITLE
Merge in Samarjeet's updated figure and code

### PR DESCRIPTION
**Pull request by davidlmobley**

### CODE REVIEW

**Comment by davidlmobley** [comment]
Grr, has conflicts because Samarjeet didn't update his TeX first. Off to meetings. @JIMonroe or @justinGilmer do you have a minute to sort it out?

**Comment by JIMonroe** [comment]
I can work on this.

**Comment by JIMonroe** [comment]
@samarjeet I just went with the most up to date tex version from master, which means you may have lost edits to the text that you made.

Could you check this and make changes if you want to?

Before you make changes, though, please run

```
git checkout post_edits
git pull post_edits
```

When you're done, commit and run `git push origin post_edits` to add you're commits here.

**Comment by davidlmobley** [comment]
Alternatively you can bring in edits to the caption in a separate PR from a separate branch, which is the approach I proposed in the toher thread, @samarjeet . (In which case we should merge this one first.)

**Comment by davidlmobley** [comment]
@samarjeet were you going to edit the caption, etc.? I want to get this merged and resubmitted.

**Comment by samarjeet** [comment]
@davidlmobley Edited. Can you please review and merge ?

File: paper/basic_training.tex [hunk 1]
```diff
@@ -1077,8 +1077,9 @@ \subsubsection{ Ewald Summation}
 
 \begin{figure}[h]
 \centering
-\includegraphics[width=\linewidth]{decay_comparison.pdf}
-    \caption{Comparison of decay of the original $r^{-1}$ term for Coulomb interactions (blue,*), the resulting direct-space term after Gaussian screening (black,-) and the $r^{-6}$ in van der Waals term (red, -.).  }
+\includegraphics[width=\linewidth]{py_decay_comparison.pdf}
+    \caption{Comparison of decay of the original $r^{-1}$ term for Coulomb interactions (blue,*), the resulting direct-space term after Gaussian screening (black,-) and the $r^{-6}$ in van der Waals term (red, -.). 
+    Note: erfc(r)/r has been scaled to show relative decay of each term. }
```
Comment (davidlmobley) [inline paper/basic_training.tex]:
```suggestion
    Note: The value on the vertical axis has been scaled to allow easy visualization of the relative decay of each term. }
```

**Comment by davidlmobley** [comment]
Thanks!

**Pull request merged**